src/test/kotlin/com/karumi/maxibonkata/KarumiHQsTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.karumi.maxibonkata
+
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.properties.forAll
+import io.kotlintest.specs.ShouldSpec
+import com.karumi.maxibonkata.Generators.DeveloperGenerator
+import com.karumi.maxibonkata.Generators.HungryDeveloperGenerator
+import com.karumi.maxibonkata.Generators.NotSoHungryDeveloperGenerator
+import io.kotlintest.properties.Gen
+
+class KarumiHQsTest : ShouldSpec() {
```

Comment:
You tested all cases for a single developer and a group of developers but not the `chat for a group` one.